src/main/java/com/xeeva/catalog/RequisitioningPage.java

```diff
@@ -479,7 +476,7 @@ public void cancelRequisitionFromRecentOrders(String comments){
 				String getStatus = driver.findElement(By.xpath("//*[@id='gvRecentOdersGrid']/tbody/"
 						+ "tr["+ row +"]/td[8]/span")).getText();
 				if(!getStatus.contains("Canceled By Requester")){
-					driver.setPageTimeout(2);
+					driver.setElementTimeout(2);
 					driver.findElement(By.xpath("//*[@id='gvRecentOdersGrid']/tbody"
 							+ "/tr["+ row +"]/td[13]/div/a/i")).jsClick();
 					//Handle Alert if present

```

Comment:
Please set the element timeout back to the default - driver.setElementTimeout(Constants.ELEMENT_TIMEOUT